lib/duration.rb

```diff
@@ -0,0 +1,37 @@
+require "date"
+
+module HotelBooking
+  class Duration
+    attr_reader :start_date, :end_date
+
+    def initialize(start_date:, end_date:)
+      @start_date = convert_to_date(start_date)
```

Comment:
Could we use `Date.parse(start_date)` instead of using the `convert_to_date` method?